ui/components/typeahead-dropdown.js

```diff
@@ -0,0 +1,44 @@
+import { Combobox } from '@headlessui/react'
+import { CheckIcon } from '@heroicons/react/solid'
+
+export default function TypeaheadDropdown({ filtered, selected }) {
+  return (
+    <>
+      {filtered.length > 0 && (
+        <Combobox.Options className='absolute -left-[13px] z-10 mt-1 max-h-60 w-56 overflow-auto rounded-md border border-gray-700 bg-gray-800 py-1 text-2xs ring-1 ring-black ring-opacity-5 focus:outline-none'>
```

Comment:
Wouldn't recommend we sever the `ComboBox` elements what if we create a component that goes inside the `Combobox.Options` (almost like a wrapper for the contents)?